pkg/cmd/grafana-cli/commands/conflict_user_command_test.go

```diff
@@ -629,11 +628,6 @@ func TestMergeUser(t *testing.T) {
 			// test starts here
 			err = r.MergeConflictingUsers(context.Background())
 			require.NoError(t, err)
-
-			// user with uppercaseemail should not exist
-			query := &models.GetUserByIdQuery{Id: userWithUpperCase.ID}
-			err = sqlStore.GetUserById(context.Background(), query)
-			require.Error(t, user.ErrUserNotFound, err)
```

Comment:
I think this was the essence of the test, so either comment it out/skip completely or maybe bring in the user.Service to check for user's existence.

Reply:
I reached out to the author of the test, the solution would be using fakes as intended